kratos/python_scripts/python_solver.py

```diff
@@ -48,6 +54,12 @@ def GetDefaultSettings(cls):
             "echo_level" : 0
         }""")
 
+    @classmethod
+    def ListOfParametersNotRecursivelyValidated(cls):
```

Comment:
this does not really make sense I would say. This would grow infinitely due to different settings
Also it is difficult to check if it works for all cases

Reply:
Well, it was the only thing I came out. The solution could be to plain the parameters..., but would make a mess of refactor and more messy to manage